cef/CustomEditField.rbvcp

```diff
@@ -34,7 +34,7 @@ Folder=Undo;CustomEditField/Undo;&hDB28231;&h258C7F1B;false
 Folder=Extras;CustomEditField/Extras;&h4281A7A3;&h258C7F1B;false
 Folder=MessageCenter;CustomEditField/MessageCenter;&h4E252217;&h258C7F1B;false
 Module=DragImage;CustomEditField/DragImage.rbbas;&h67E17DBB;&h258C7F1B;false
-Folder=imgs;imgs;&h505F6EB;&h258C7F1B;false
+Folder=imgs;CustomEditField/imgs;&h505F6EB;&h258C7F1B;false
```

Comment:
This one is wrong. It's a bug in Xojo that's causing this. The path needs to remain simply "imgs", not "CustomEditField/imgs". Please manually fix this and commit/push

Reply:
Nevermind - I'll simply move the imgs folder on disk to the subfolder and commit the change, so that it's mentioned in the log separately